src/components/Button/Button.js

```diff
@@ -0,0 +1,93 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import ButtonType from './ButtonType';
+import './Button.scss';
+
+const propTypes = {
+    /** Content of the button, e.g. a string label or more complex React components */
```

Comment:
I'm not sure we need the comments here. The naming of the props should be enough to indicate how/where they are used.

Reply:
Removed